frontend/src/pages/CourseDashboard/index.js

```diff
@@ -0,0 +1,218 @@
+    user && user.homeRegionId && user.homeRegionId === 14
+  ), [user]);
+
+  const getFiltersWithAllRegions = () => {
+    const filtersWithAllRegions = [...allRegionsFilters];
+    return filtersWithAllRegions;
+  };
+  const centralOfficeWithAllRegionFilters = getFiltersWithAllRegions();
+
+  const defaultFilters = useMemo(() => {
+    if (hasCentralOffice) {
+      return [...centralOfficeWithAllRegionFilters,
+        {
+          id: uuidv4(),
+          topic: 'startDate',
+          condition: 'is within',
+          query: defaultDate,
+        }];
+    }
+
+    return [
+      {
+        id: uuidv4(),
+        topic: 'region',
+        condition: 'is',
+        query: defaultRegion,
+      },
+      {
+        id: uuidv4(),
+        topic: 'startDate',
+        condition: 'is within',
+        query: defaultDate,
+      },
+    ];
+  }, [defaultRegion, hasCentralOffice, centralOfficeWithAllRegionFilters]);
+
+  const [filters, setFiltersInHook] = useSessionFiltersAndReflectInUrl(
+    FILTER_KEY,
+    defaultFilters,
+  );
+
+  const setFilters = useCallback((newFilters) => {
+    setFiltersInHook(newFilters);
+    setResetPagination(true);
+  }, [setFiltersInHook]);
+
+  // Remove Filters.
+  const onRemoveFilter = (id, addBackDefaultRegions) => {
+    const newFilters = [...filters];
+    const index = newFilters.findIndex((item) => item.id === id);
+    if (index !== -1) {
+      newFilters.splice(index, 1);
+      if (addBackDefaultRegions) {
+        // We always want the regions to appear in the URL.
+        setFilters([...allRegionsFilters, ...newFilters]);
+      } else {
+        setFilters(newFilters);
+      }
+    }
+  };
+
+  // Apply filters.
+  const onApplyFilters = (newFilters, addBackDefaultRegions) => {
+    if (addBackDefaultRegions) {
+      // We always want the regions to appear in the URL.
+      setFilters([
+        ...allRegionsFilters,
+        ...newFilters,
+      ]);
+    } else {
+      setFilters([
+        ...newFilters,
+      ]);
+    }
+
+    ariaLiveContext.announce(`${newFilters.length} filter${newFilters.length !== 1 ? 's' : ''} applied to courses with activity reports `);
+  };
+
+  const filtersToApply = useMemo(() => expandFilters(filters), [filters]);
+
+  useDeepCompareEffect(() => {
+    async function fetchCourseData() {
+      setIsLoading(true);
+      // Filters passed also contains region.
+      const filterQuery = filtersToQueryString(filtersToApply);
+      try {
+        const data = await fetchCourseDashboardData(
+          filterQuery,
+        );
+        console.log('data', data);
+        setCourseData(data);
+        updateError('');
+      } catch (e) {
+        updateError('Unable to fetch course data');
+      } finally {
+        setIsLoading(false);
+      }
+    }
+    // Call courses fetch.
+    fetchCourseData();
+  }, [
+    filtersToApply,
+  ]);
+
+  return (
+    <div className="ttahub-course-dashboard">
+      <Helmet>
+        <title>iPD Courses</title>
```

Comment:
This doesn't align with all the other page title conventions. I think we want them to look similar so they are easier to read in google analytics

Reply:
I wouldn't worry about this actually, it matches what's in the h1 and I think we want Lauren to decide the page title since she's the one in Google Analytics.